scripts/add_electricity.py

```diff
@@ -537,6 +614,8 @@ def add_nice_carrier_names(n, config=None):
     attach_extendable_generators(n, costs, ppl)
 
     estimate_renewable_capacities(n)
+    attach_OPSD_renewables(n)
```

Comment:
```suggestion
    estimate_renewable_capacities(n)
    
    attach_OPSD_renewables(n)
```

Reply:
estimate_renewable_capacities retrieves installed capacities on country basis (also from OPSD) and distributes them according to a heuristic based on land availability and capacity factor.
attach_OPSD_renewables takes the countries where lat/lon are given and overwrites the capacities in this country.

in my opinion, this belongs together. don't you think? But I mean it's just one line, so whatever...

Reply:
We can leave it together :)